source/views/sis/student-work/detail.android.js

```diff
@@ -135,6 +135,8 @@ function LastUpdated({when}: {when: string}) {
         Last updated:
         {' '}
         {moment(when, 'YYYY/MM/DD').calendar()}
+        {'\n'}
+        Powered by St. Olaf Student Employment job postings
```

Comment:
Could provide a link to https://www.stolaf.edu/apps/stuwork/index.cfm?fuseaction=SearchResults or https://www.stolaf.edu/apps/stuwork/

Reply:
Can we revisit this when we actually have the links working on the others, and have restyled them?

Reply:
I guess I don't see why hesitate when the [one for calendar](https://github.com/StoDevX/AAO-React-Native/pull/1614/files#diff-7ba76014bd6f407012e30d12c4aa7213R16) was added for this PR?

Reply:
because this one isn't actually using ListFooter. I can make it use ListFooter, but I just re-used the existing footer styles in here.

Reply:
Alright, that's fine.